src/app/Dashboard/LayoutTemplatePicker.tsx

```diff
@@ -0,0 +1,633 @@
+          </DrawerActions>
+        </DrawerHead>
+        <DrawerPanelBody style={{ marginTop: '-3.5em' }}>
+          {selectedTemplate ? (
+            <DescriptionList isFillColumns>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Name</DescriptionListTerm>
+                <DescriptionListDescription>{selectedTemplate?.name}</DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Description</DescriptionListTerm>
+                <DescriptionListDescription>{selectedTemplate?.description}</DescriptionListDescription>
+              </DescriptionListGroup>
+              {selectedTemplate?.vendor && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Vendor</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.vendor}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              {selectedTemplate?.version && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Version</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.version}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              <DescriptionListGroup>
+                <DescriptionListTerm>Preview ({numCards} Cards)</DescriptionListTerm>
+                {
+                  <ServiceContext.Provider value={fakeServices}>
+                    <ChartContext.Provider value={fakeChartContext}>
+                      <Grid
+                        id={'dashboard-layout-preview-grid'}
+                        hasGutter
+                        onClick={(e) => {
+                          e.preventDefault();
+                          e.stopPropagation();
+                        }}
+                      >
+                        {numCards === 0 ? (
+                          <EmptyState>
+                            <EmptyStateBody>Empty preview</EmptyStateBody>
+                          </EmptyState>
+                        ) : (
+                          <>
+                            {selectedTemplate.cards
+                              .slice(0, CARD_PREVIEW_LIMIT)
+                              .filter((cfg) => hasConfigByName(cfg.name))
+                              .map((cfg, idx) => (
+                                <GridItem span={numCards === 1 ? 12 : 6} key={idx} order={{ default: idx.toString() }}>
+                                  {/* TODO: remove this once we have a preview for JFRMetricsChartCard */}
+                                  {cfg.name === 'JFRMetricsChartCard' ? (
+                                    <Card isFullHeight isCompact>
+                                      <CardHeader>
+                                        <Title headingLevel={'h4'}>{cfg.props['chartKind']}</Title>
```

Comment:
Seems like the empty card title is not receiving the css.

![Screenshot from 2023-04-12 20-56-01](https://user-images.githubusercontent.com/68053619/231618139-0b521218-a001-4cae-a5a4-9d274243386b.png)

Reply:
Try this?

```css

#dashboard-layout-preview-grid .pf-c-card__title {
  --pf-c-card__title--FontSize: 0.7em;
}

#dashboard-layout-preview-grid .pf-c-title {
  font-size: 0.7em;
  font-weight: 700;
}
```